lib/srv/desktop/rdp/rdpclient/src/cliprdr.rs

```diff
@@ -548,6 +545,22 @@ impl<T: FormatName> FormatListPDU<T> {
     }
 }
 
+fn encode_clipboard(mut data: Vec<u8>) -> RdpResult<(Vec<u8>, ClipboardFormat)> {
+    if data.is_ascii() {
+        if matches!(data.last(), Some(x) if *x != 0) {
+            data.push(0);
```

Comment:
Can we add a comment just so it's clear that this is adding a 1-byte null terminator?